src/js/controller/controller.js

```diff
@@ -776,7 +776,7 @@ Object.assign(Controller.prototype, {
             if (state === STATE_ERROR) {
                 return;
             }
-            if (state === STATE_PLAYING && _model.get('playReason') !== meta.reason) {
+            if (meta && state === STATE_PLAYING && _model.get('playReason') !== meta.reason) {
```

Comment:
I noticed that there were console errors related to  `TypeError: Cannot read property 'reason' of undefined` and adding this conditional seems to address this behavior.